plugins/connection/aws_ssm.py

```diff
@@ -534,7 +538,11 @@ def _flush_stderr(self, subprocess):
 
     def _get_url(self, client_method, bucket_name, out_path, http_method, profile_name, extra_args=None):
         ''' Generate URL for get_object / put_object '''
-        region_name = self.get_option('region') or 'us-east-1'
+        region_name = self.get_option('bucket_region')
```

Comment:
I believe you can write this like that: `region_name = self.get_option('bucket_region') or self.get_option('region') or 'us-east-1'`